activerecord/lib/active_record/connection_adapters/postgresql/oid/uuid.rb

```diff
@@ -13,8 +13,9 @@ def type
             :uuid
           end
 
-          def cast(value)
-            value.to_s[ACCEPTABLE_UUID, 0]
+          def cast_value(value)
```

Comment:
Can you make the `cast_value` method private?

https://github.com/rails/rails/blob/5ca19efafea08984ff1c464c64969f82b9db3569/activemodel/lib/active_model/type/value.rb#L116-L123

Reply:
Can you fix the style issue?

https://codeclimate.com/github/rails/rails/pull/35263

![image](https://user-images.githubusercontent.com/12642/53067077-f0ad8600-3515-11e9-9f66-dd6701eced1b.png)